source/jormungandr/jormungandr/scenarios/helper_classes/helper_utils.py

```diff
@@ -752,16 +752,24 @@ def timed_logger(logger, task_name, request_id):
         yield logger
     finally:
         end = time.time()
-        elapsed_time = end - start
+        elapsed_time = (end - start) * 1000
         start_in_ms = int(start * 1000)
         end_in_ms = int(end * 1000)
+        # if elapsed_time is smaller than 0.5ms, we consider that it's negligible
+        if elapsed_time < 0.5:
+            logger.info(
+                "Task : {}, request : {} finished quasi-instantly".format(
+                    task_name,
+                    request_id,
+                )
+            )
+            return
+
```

Comment:
is it really worthwhile to change the log message in this case ?

Reply:
you're right... kind of over-engineering....